rust/src/measurements/make_private_lazyframe/group_by/make_private_group_by.bib

```diff
@@ -0,0 +1,9 @@
+@misc{rogers2023unifyingprivacyanalysisframework,
+    title={A Unifying Privacy Analysis Framework for Unknown Domain Algorithms in Differential Privacy}, 
+    author={Ryan Rogers},
+    year={2023},
+    eprint={2309.09170},
+    archivePrefix={arXiv},
+    primaryClass={cs.CR},
+    url={https://arxiv.org/abs/2309.09170}, 
```

Comment:
Should there be a link rendered in the PDF? It just seems to be plain text.

Reply:
Annoying that latex doesn't handle this automatically. I don't immediately see a setting to enable it.